lib/tarantool_server.py

```diff
@@ -1016,16 +1016,6 @@ def stop(self, silent=True, signal=signal.SIGTERM):
             color_log('Server [%s] start against running ...\n',
                       schema='test_var')
             return
-        if self.status != 'started':
-            if not silent:
-                raise Exception('Server is not started')
-            else:
-                color_log(
-                    'Server [{0.name}] is not started '
-                    '(status:{0.status}) ...\n'.format(self),
-                    schema='test_var'
-                )
-            return
```

Comment:
Looks as a fix of (or tightly related to) #256.

Reply:
Yep, looks like a fix. Mentioned in the commit message.